zerver/lib/subscription_info.py

```diff
@@ -441,9 +448,10 @@ def has_metadata_access_to_previously_subscribed_stream(
 def gather_subscriptions_helper(
     user_profile: UserProfile,
     include_subscribers: bool = True,
+    include_archived_streams: bool = False,
 ) -> SubscriptionInfo:
     realm = user_profile.realm
-    all_streams = get_active_streams(realm).values(
+    all_streams = get_all_streams(realm, include_archived_streams).values(
```

Comment:
Spell this as `get_all_streams(realm, include_archived_streams=include_archived_streams)`: passing via kwargs makes it obvious to someone reading this code what that second parameter does.